backend/src/apis/fetch.rs

```diff
@@ -115,6 +115,7 @@ mod tests {
     use super::*;
 
     #[test]
+    #[ignore]
```

Comment:
If I remove the ignore and add an .env file with `COVALENT_API_KEY=cqt_rQYxVkytpYCwgC9XmPrkBq8HgVk6:`
I get this tests working. 

Can you try reproducing it on your machine? The only reason that it might be failing is that they enforce some check on the IP that uses the API, so maybe it's worth also trying using a different API KEY